src/main/java/org/takes/rq/CapInputStream.java

```diff
@@ -34,6 +34,10 @@
  * @author Yegor Bugayenko (yegor@teamed.io)
  * @version $Id$
  * @since 0.16
+ * @todo #254:30min CapInputStream should delegate all standard InputStream
```

Comment:
@pinaf hmm, I thought I answered to this... my comment is missing. The idea is to remove temp files on body stream close. There should quite common logic - after reading request body there should `close` on the stream However, in some cases body InputStream can be wrapped with CapInputStream - so in this case we just get IOException which is wrong. That's why I created this puzzle.